### DISCUSSION TITLE
Contributions to sbt

### DISCUSSION THREAD

**Discussion by gontard** [sbt/sbt#7029] [Q&A] [answered]
Hello,
i have created two pull requests (https://github.com/sbt/sbt/pull/6824, https://github.com/sbt/sbt/pull/6903). They have been approved and merged in the `develop` branch. Also an issue https://github.com/sbt/sbt/issues/5504 was resolved by one of these PRs.
However they weren't included in recent releases ([1.7.0](https://github.com/sbt/sbt/releases/tag/v1.7.0), [1.7.1](https://github.com/sbt/sbt/releases/tag/v1.7.1)). 

I am wondering why. 
How PR are pick up from develop to the release branch? Which/How PRs are selected?

Thanks :)

**Comment by He-Pin** [comment]
May be develep branch target 1.8？

**Comment by He-Pin** [comment]
How about target 1.7.x and make pull request again？

**Comment by gontard** [comment]
@He-Pin Thanks for your suggestions but i would like to understand clearly how it's works. 
Maybe it's documented somewhere 🤷 but i haven't found anything in the CONTRIBUTING guide. 

I would like to get an authoritative answer from a `sbt` maintainer.

**Reply by He-Pin** [reply]
Sorry when replay it I am using a mobile phone at airport.
I think the sbt maintainer should make that more cleaner at the CONTRIBUTING guide

**Comment by adpi2** [comment, marked as answer]
As far as I know, the `develop` branch will be sbt 2.x, with full support for Scala 3. For your PRs to be released in the next minor version, 1.8.0, you can backport them to the current default branch, which is `1.8.x`.

**Reply by gontard** [reply]
Thanks for your answer.
I have opened (#7030 and #7031) to backport on 1.8.x

**Reply by eed3si9n** [reply]
Yea. `develop` branch is somewhat experimental at this point because it's targeting sbt 2.x, so yea for more stable changes I should've directed you to 1.8.x and 1.7.x lines.

**Reply by gontard** [reply]
> I should've directed you to 1.8.x and 1.7.x lines.

No problem. 
Thanks for your answer and all your work on sbt.
